cases/polygon-rendering/main.js

```diff
@@ -0,0 +1,257 @@
+/* eslint-disable no-console */
+import GUI from 'lil-gui';
+import GeoJSON from 'ol/format/GeoJSON.js';
+import Layer from 'ol/layer/Layer.js';
+import Link from 'ol/interaction/Link.js';
+import Map from 'ol/Map.js';
+import VectorLayer from 'ol/layer/Vector.js';
+import VectorSource from 'ol/source/Vector.js';
+import View from 'ol/View.js';
+import WebGLVectorLayerRenderer from 'ol/renderer/webgl/VectorLayer.js';
+import {useGeographic} from 'ol/proj.js';
+
+useGeographic();
+
+const map = new Map({
+  layers: [],
+  target: 'map',
+  view: new View({
+    center: [0, 0],
+    zoom: 4,
+  }),
+});
+
+const source = new VectorSource();
+
+const colors = ['#6ff05b', '#00AAFF', '#faa91e'];
+
+const gui = new GUI();
+
+const link = new Link();
+
+const style = {
+  'fill-color': ['get', 'color'],
+  'stroke-color': 'gray',
+  'stroke-width': 0.5,
+};
+
+const gui_obj = {
+  'Feature count': 200000,
+  'Use WebGL': false,
+  'Vertices': 4,
+  'Outline': true,
+};
+
+const useWebGLCheckbox = gui.add(gui_obj, 'Use WebGL');
+const featureCountSlider = gui.add(gui_obj, 'Feature count', 100000, 500000);
+const verticesCount = gui.add(gui_obj, 'Vertices', 3, 20, 1);
+const outlineCheckbox = gui.add(gui_obj, 'Outline');
+
+useWebGLCheckbox.onChange((/** @type {boolean} */ value) => {
+  if (value) {
+    link.update('renderer', 'webgl');
+    useWebGL();
+  } else {
+    link.update('renderer', 'canvas');
+    useCanvas();
+  }
+});
+
+featureCountSlider.onFinishChange(() => {
+  link.update('count', featureCountSlider.getValue());
+  resetData(
+    parseInt(featureCountSlider.getValue()),
+    parseInt(verticesCount.getValue())
+  );
+});
+
+verticesCount.onFinishChange(() => {
+  link.update('vertices', verticesCount.getValue());
+  resetData(
+    parseInt(featureCountSlider.getValue()),
+    parseInt(verticesCount.getValue())
+  );
+});
+
+outlineCheckbox.onChange((/** @type {boolean} */ value) => {
+  link.update('outline', value ? 'yes' : 'no');
+  if (value) {
+    style['stroke-width'] = 0.5;
+  } else {
+    delete style['stroke-width'];
```

Comment:
This is not enough to get the stroke to disappear; you also need to remove the `stroke-color` property

Reply:
Made the necessary updates to remove and add `stroke-color` property. @jahow